src/screens/DataChart.js

```diff
@@ -32,10 +32,7 @@ const Examples = () => (
       background="light-3"
     >
       <Paragraph margin="none">
```

Comment:
I think we can just remove this whole Paragraph. It feels a bit misleading to include an API surface change in a non-major version without explaining the "beta" concept.